elastic/kibana
# View request flyout

Fixes: https://github.com/elastic/kibana/issues/110926 and https://github.com/elastic/kibana-team/issues/435

### Summary

This PR adds a generic view request flyout to es_ui_shared that can be used anywhere we want to expose an ES API requests to the user. Allowing the user to also copy the request to their clipboard and to view the request in Console.

I've also implemented the modal in the request preview in the ingest pipelines edit screen.

**Note**: There are more of these modals throughout our apps (CCR, remote clusters, ILM, watcher and rollups) I decided to implement it only for `Ingest Pipelines` to keep the PR more concise and will create another small PR after for patching that up.

**How to test**
1. Start elasticsearch and kibana
2. Navigate to `Stack Management` -> `Ingest pipelines`
3. Edit an existing pipeline or create a new one
4. Verify that the show request button works and flyout opens
5. Verify that the copy to clipboard works and that the show in console works also

**Screenshots**

<details>
<summary>before/after</summary>

// Before
![Screenshot 2022-03-10 at 15 24 19](https://user-images.githubusercontent.com/1191206/157681903-8db7ff5b-a53d-4ffa-a6e2-1e9b7f1f09a2.png)


// After
![Screenshot 2022-03-10 at 15 22 09](https://user-images.githubusercontent.com/1191206/157681669-e802bfc4-8627-4015-8df1-07a4ee6cf93c.png)

</details>
